src/main/webapp/app/entities/programming-exercise/status/programming-exercise-status.module.ts

```diff
@@ -0,0 +1,23 @@
+import { CUSTOM_ELEMENTS_SCHEMA, NgModule } from '@angular/core';
+import { JhiLanguageService } from 'ng-jhipster';
+import { JhiLanguageHelper } from 'app/core/language/language.helper';
+import { ProgrammingExerciseInstructorExerciseStatusComponent, ProgrammingExerciseInstructorStatusComponent } from './';
+import { ArtemisParticipationModule } from 'app/entities/participation/participation.module';
+import { ArtemisSharedModule } from 'app/shared';
+
+@NgModule({
+    imports: [ArtemisSharedModule, ArtemisParticipationModule],
+    declarations: [ProgrammingExerciseInstructorStatusComponent, ProgrammingExerciseInstructorExerciseStatusComponent],
+    providers: [{ provide: JhiLanguageService, useClass: JhiLanguageService }],
+    exports: [ProgrammingExerciseInstructorStatusComponent, ProgrammingExerciseInstructorExerciseStatusComponent],
+    schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
Is the `CUSTOM_ELEMENTS_SCHEMA` used on purpose here?
```suggestion
```

Reply:
You are right, I forgot to remove it :see_no_evil: